appveyor.yml

```diff
@@ -1,21 +1,47 @@
 environment:
   matrix:
-  # Releases
-   - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x64/0.5/julia-0.5-latest-win64.exe"
-  # Nightlies
+  - JULIAVERSION: "julialang/bin/winnt/x86/0.5/julia-0.5-latest-win32.exe"
```

Comment:
Example.jl got missed from the first round of PR's to change them everywhere, the script `@staticfloat` used didn't tell him when some PR's failed to open due to the github rate limit